Citizen-Science/src/screens/SignupScreen.tsx

```diff
@@ -0,0 +1,118 @@
+
+                {/* Second Blob Cluster */}
+                <Blobs rotationDeg={'0deg'} widthPercentage={20} heightPercentage={10} position={{ top: 6, left: 80 }} />
+                <Blobs rotationDeg={'0deg'} widthPercentage={6} heightPercentage={3} position={{ top: 15, left: 93 }} />
+
+                {/* Logo name */}
+                <LogoName position={'bottomRight'} color={'grey'}/>
+
+                {/* Back Button */}
+                <BackButton />
+
+                {/* 'Create an Account' title on signup */}
+                <ScreenTitle
+                    text={'Create an \nAccount'}
+                    textStyle={'title'}
+                    fontSize={5}
+                    color={'white'}
+                    // Uses responsive library {width, height} through the components file
+                    position={{top: 20 , left: 5}}
+                />
+
+                {/* Name TextField */}
+                <CustomTextField
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    placeholder="Name"
+                    value={email}
+                    onChangeText={setName}
+                    borderColor="#5EA1E9"
+                    borderRadius={10}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(60) }}
+                />
+
+                {/* Email TextField */}
+                <CustomTextField
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    // TODO: Add option to signup with phone number
+                    placeholder="Email"
+                    value={email}
+                    onChangeText={setEmail}
+                    borderColor="#5EA1E9"
+                    borderRadius={10}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(69) }}
+                />
+
+
+                {/* Password TextField */}
+                <CustomTextField
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    placeholder="Password"
+                    value={password}
+                    onChangeText={setPassword}
+                    secureTextEntry={true} // To hide password input
+                    borderColor="#5EA1E9"
+                    borderRadius={10}
+                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(78) }}
+                />
+
+                {/* Signup Button */}
+                <CustomButton
+                    size={{width: responsiveWidth(70), height: responsiveHeight(5.5)}}
+                    label="Sign Up"
+                    labelColor="white"
+                    backgroundColor="#5EA1E9"
+                    //TODO potentially redirect to home page after account creation
+                    onPress={() => navigation.navigate('Login')}
```

Comment:
I'd recommend `onPress={() => navigation.replace('Login')}`
instead of navigate since it removes the Sign Up screen from the history. Screens like Sign Up shouldn't be in history since we don't want people to go back to sign up after already doing so